inference-engine/src/mkldnn_plugin/CMakeLists.txt

```diff
@@ -207,15 +203,8 @@ target_include_directories(${TARGET_NAME}_obj PRIVATE $<TARGET_PROPERTY:inferenc
                                                       $<TARGET_PROPERTY:inference_engine_transformations,INTERFACE_INCLUDE_DIRECTORIES>
                                                       $<TARGET_PROPERTY:openvino::itt,INTERFACE_INCLUDE_DIRECTORIES>)
 
-if(USE_CNNNETWORK_LPT)
-    target_include_directories(${TARGET_NAME}_obj PRIVATE
-        $<TARGET_PROPERTY:inference_engine_lp_transformations_legacy,INTERFACE_INCLUDE_DIRECTORIES>)
-    target_compile_definitions(${TARGET_NAME}_obj PRIVATE
-        $<TARGET_PROPERTY:inference_engine_lp_transformations_legacy,INTERFACE_COMPILE_DEFINITIONS>)
-else()
-    target_include_directories(${TARGET_NAME}_obj PRIVATE
+target_include_directories(${TARGET_NAME}_obj PRIVATE
```

Comment:
let's merge with the previous `target_include_directories`

Reply:
done, thanks